packages/components/src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx

```diff
@@ -11,34 +12,53 @@ const ContentPageHeader = ({
   buttonLabel,
   buttonUrl,
   site,
+  image,
   LinkComponent,
 }: ContentPageHeaderProps) => {
+  const hasImage = !!image?.src
+
   return (
     <div className="bg-brand-canvas text-base-content-strong">
-      <div className="mx-auto flex max-w-screen-xl flex-col gap-8 px-6 py-8 md:px-10">
-        <div className="flex max-w-[54rem] flex-col">
-          <Breadcrumb links={breadcrumb.links} LinkComponent={LinkComponent} />
-          <div className="mt-8 flex flex-col gap-5 md:mt-6">
-            <h1 className="prose-display-lg break-words">{title}</h1>
-            <p className="prose-title-lg-regular">{summary}</p>
-          </div>
-          {buttonLabel && buttonUrl && (
-            <div className="mt-9">
-              <LinkButton
-                href={getReferenceLinkHref(
-                  buttonUrl,
-                  site.siteMap,
-                  site.assetsBaseUrl,
-                )}
-                LinkComponent={LinkComponent}
-                isWithFocusVisibleHighlight
-              >
-                {buttonLabel}
-              </LinkButton>
+      <div className="mx-auto flex max-w-screen-xl flex-col gap-8">
+        <div className="grid-rows-[1fr fit-content] grid gap-10 lg:grid-cols-12">
+          <div
+            className={`flex max-w-[54rem] flex-col px-6 py-8 max-md:row-span-${hasImage ? 1 : 2} md:px-10 lg:col-span-${hasImage ? 7 : 12}`}
+          >
+            <Breadcrumb
+              links={breadcrumb.links}
+              LinkComponent={LinkComponent}
+            />
+            <div className="mt-8 flex flex-col gap-5 md:mt-6">
+              <h1 className="prose-display-lg break-words">{title}</h1>
+              <p className="prose-title-lg-regular">{summary}</p>
             </div>
+            {buttonLabel && buttonUrl && (
+              <div className="mb-8 mt-9">
+                <LinkButton
+                  href={getReferenceLinkHref(
+                    buttonUrl,
+                    site.siteMap,
+                    site.assetsBaseUrl,
+                  )}
+                  LinkComponent={LinkComponent}
+                  isWithFocusVisibleHighlight
+                >
+                  {buttonLabel}
+                </LinkButton>
+              </div>
+            )}
+            <div className="prose-body-sm text-base-content-subtle">{`Last updated ${getFormattedDate(lastUpdated)}`}</div>
```

Comment:
issue: I think we lost the gap between the summary and the last updated date:

![image](https://github.com/user-attachments/assets/1ae7beaf-c3de-4557-a76f-6613b731c391)

Reply:
yeap, fixed these!